buildenv/jenkins/common/build.groovy

```diff
@@ -845,7 +845,7 @@ def build_all() {
                 timeout(time: 5, unit: 'HOURS') {
                     if ("${DOCKER_IMAGE}") {
                         // TODO: remove this workaround when https://github.com/adoptium/infrastructure/issues/3597 resolved. related: infra 9292
-                        if ((SDK_VERSION.toInteger() >= 17 && PLATFORM ==~ /x86-64_linux.*/) || (PLATFORM ==~ /ppc64le_linux.*/ )) {
+                        if ((PLATFORM ==~ /x86-64_linux.*/ && (SDK_VERSION.toLowerCase() == 'next' || SDK_VERSION.toInteger() >= 17)) || (PLATFORM ==~ /ppc64le_linux.*/ )) {
```

Comment:
I don't think we need or want `.toLowerCase()` here. As well, there are missing parentheses, the line is excessively long, and the space before `))` is unwanted.

Reply:
the `SDK_VERSION` comes from input params, so `.toLowerCase()` will guarantee all possible cases in case someone re-run or try to run directly. The other concerned covered. Thanks.